README.md

```diff
@@ -254,6 +254,44 @@ const {verify} = assertionMethodPublicKey.verifier();
 `publicMethodFor` will return `undefined` if no key is found for a given
 purpose.
 
+### Backwards Compatibility with the 2018/2019 Crypto Suites
+
+By default, this did key driver returns DID Documents that have the 2020
+crypto suites for verification and key agreement.
+If you need DID Documents that are using the 2018/2019 crypto suites,
+you can customize the driver as follows.
+
+```js
+const didKeyDriver2018 = driver({
+ verificationSuite: Ed25519VerificationKey2018
```

Comment:
Should that value be a string or a var imported from elsewhere?

Reply:
Ah, yeah, I'll add the import.

Reply:
Added